### PR TITLE
update language files

### PR DESCRIPTION
**Pull request by mediaminister**
This pull request includes:
- update language files for translators
- add language test

### CODE REVIEW

**Comment by dagwieers** [comment]
Oh damn, I just made almost the exact same PR :-/

File: resources/language/resource.language.nl_nl/strings.po [hunk 1]
```diff
@@ -304,4 +307,4 @@ msgstr "Tijdelijke download folder"
 
 msgctxt "#30911"
 msgid "Update frequency in days"
-msgstr ""
+msgstr "Update frequentie in dagen"
```
Comment (dagwieers) [inline resources/language/resource.language.nl_nl/strings.po]: Should that not be one word? *Updatefrequentie*

**Comment by mediaminister** [comment]
You can close this and merge yours.

File: resources/language/resource.language.nl_nl/strings.po (hunk 1 above)
Comment (mediaminister) [reply resources/language/resource.language.nl_nl/strings.po]: Yes.

**Comment by dagwieers** [comment]
No no, you were first ;-)

**Comment by mediaminister** [comment]
Okay, enough for today, tomorrow I will see what still needs to be done.

**Comment by mediaminister** [comment]
If this can be merged I can open a "Help test and translate InputStream Helper v0.4.3" issue.

**Comment by dagwieers** [comment]
Sure, this is fine for me. You can merge as well ;-)

**Pull request merged**